_data/examples.json

```diff
@@ -383,6 +383,19 @@
         "name": "layer_falkensee",
         "title": "Line Chart with Highlighted Rectangles",
         "style": "background-size: auto 90%; background-position-y: center !important;"
+      },
+      {
+        "name": "layer_line_mean_point_raw",
+        "title": "Layering mean values over raw values"
+      },
+      {
+        "name": "layer_line_rolling_mean_point_raw",
+        "title": "Layering rolling mean values over raw values"
+      },
+      {
+        "name": "layer_likert",
+        "title": "Likert Scale Ratings Distributions and Medians",
+        "description": "Likert Scale Ratings Distributions and Medians. (Figure 9. from @jhoffswell and @zcliu's 'Interactive Repair of Tables Extracted from PDF Documents on Mobile Devices' -- http://idl.cs.washington.edu/files/2019-InteractiveTableRepair-CHI.pdf)"
```

Comment:
Duplicate with description. 

```suggestion
```

Reply:
It's not a duplicate, until someone fixes https://github.com/vega/vega-lite/issues/4684

Reply:
Let's fix it then. It's easy. I can do it after this is merged.

Reply:
Then move all descriptions into examples in that PR.  It doesn't make sense to do it here as you haven't done it yet.